src/ralph/licences/models.py

```diff
@@ -73,13 +74,14 @@ class Licence(
             "Should be like 'per processor' or 'per machine' and so on. ",
         ),
     )
-    # property_of = models.ForeignKey(
-    #     settings.AUTH_USER_MODEL,
-    #     on_delete=models.PROTECT,
-    #     null=True,
-    # )
-    software_category = models.ForeignKey(
-        SoftwareCategory,
+    property_of = models.ForeignKey(
+        'back_office.AssetHolder',
```

Comment:
wouldn't it be better to define `AssetHolder` in assets app? it'll be used in BO, DC, Domains and maybe somewhere else in the future